NestJS/src/Templates/template.listener.ts

```diff
@@ -5,46 +5,52 @@ import { TemplateService } from './template.service';
 const connection = new Amqp.Connection("amqp://localhost");
 const exchange = connection.declareExchange("exchangeForTemplate", 'direct', { durable: false });
 const queue = connection.declareQueue('template');
-
 @Controller('template')
 export class TemplateListener {
     constructor(private readonly templateService: TemplateService){
       this.listenQueue();
     }
 
     public distributionTasks(task){
-      if (task.title == "CREATE")this.createTemplate(task);
-      else if(task.title == "UPDATE")this.updateTemplate(task);
-      else if(task.title == "DELETE")this.deleteTemplate(task);
-      else if(task.title == "FIND_ONE")this.findOneTemplate(task);
-      else if(task.title == "FIND_ALL")this.findAllTemplate(task);
+      const obj = {
+        CREATE: this.createTemplate,
+        UPDATE: this.updateTemplate,
+        DELETE: this.deleteTemplate,
+        FIND_ALL: this.findAllTemplate,
+        FIND_ONE: this.findOneTemplate,
+      };
+      if (obj[task.title] != null) {
```

Comment:
not existing property will give you `undefined` instead of `null`
so you can simplify this to

`if (obj[task.title]) {`